### PR TITLE
Bump Redux dependency

### PR DESCRIPTION
**Pull request by dlabrecq**
Bump Redux dependency

https://issues.redhat.com/browse/COST-2906

### CODE REVIEW

**Review by dlabrecq — COMMENTED** [review]
Self

File: src/utils/omit.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Based on https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
```
Comment (gitdallas) [inline src/utils/omit.ts:2]: Omit is part of typescript 3.5+, I don't think we need this.
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-5.html#the-omit-helper-type

File: src/utils/omit.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+export type DistributiveOmit<T, K extends keyof T> = T extends unknown ? Omit<T, K> : never;
```
Comment (gitdallas) [inline src/utils/omit.ts:4]: not being used?

**Review by gitdallas — CHANGES_REQUESTED** [review]

File: src/utils/omit.ts (hunk 1 above)
Comment (dlabrecq) [reply src/utils/omit.ts]: Updated

**Review by gitdallas — APPROVED** [review]

**Pull request merged**